src/panels/lovelace/hui-root.js

```diff
@@ -204,8 +205,11 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
         view.config = viewConfig;
         view.columns = this.columns;
       }
+      if (viewConfig.background) background = viewConfig.background;
     }
 
+    this.$.view.style.background = background;
```

Comment:
view is a span. it's height is defined based on the content. I would expect people to want to style the whole background, regardless of the height of the content?

Reply:
Ah, that was addressed in #1403